src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/VetFieldEnum.java

```diff
@@ -75,30 +73,55 @@ public String getColumnValue(final VariantContext variant) {
         // TODO sci notation?
         public String getColumnValue(final VariantContext variant) {
             String out = getAttribute(variant, GATKVCFConstants.AS_RAW_RMS_MAPPING_QUALITY_KEY, null);
-            if (out == null) {
-                throw new UserException("Cannot be missing required value for alternate_bases.AS_RAW_MQ");
+            String outNotAlleleSpecific = getAttribute(variant, GATKVCFConstants.RAW_MAPPING_QUALITY_WITH_DEPTH_KEY, null);
+            String outNotAlleleSpecificAndOld = getAttribute(variant, GATKVCFConstants.RAW_RMS_MAPPING_QUALITY_DEPRECATED, null);
+            if (out == null && outNotAlleleSpecific == null && outNotAlleleSpecificAndOld == null) {
+                throw new UserException("Cannot be missing required value for alternate_bases.AS_RAW_MQ, RAW_MQandDP or RAW_MQ.");
             }
-            if (!out.endsWith("|0.00")) {
-                logger.warn("Expected AS_RAW_MQ value to end in |0.00. value is: " + out + " for variant " + variant.toString());
+            if (out != null) {
+                if(!out.endsWith("|0.00")) {
+                    logger.warn("Expected AS_RAW_MQ value to end in |0.00. value is: " + out + " for variant " + variant.toString());
+                }
+                    out = out.substring(0, out.length() - 5);
```

Comment:
we need to replace the line above with this since we might have actual data in the last field:
out = out.substring(0, out.lastIndexOf("|"));